### PR TITLE
New design notifications news

### LINKED ISSUE #1550: [Notification Subscription] Update UI For The Notifications Functionality
### Which area(s) of Podkrepi.bg are affected?

Notification Subscription

### Describe the Need

Need to update the current simplistic subscription forms (screenshots here) with the new improved design from Milena's comment: https://github.com/podkrepi-bg/frontend/issues/1550#issuecomment-1756087392

Current(old) design for reference:
 - Non-logged users - General subscribe
image
image
image

NOTE: No one will go to the footer to subscribe. We should add some pop-up that will show up when a visitor lands on the home page. It should contain an input field for the visitor to subsribe his/her email. So probably there should be a design for that one as well

 - Non-logged users - Campaign subscribe
 
image
image
image

- Confirm Subscribe Email
image

- Subscribe/Unsubscribe confirmation flow
Loader

 - Subscribe Success / Fail
image
image

-Unsubscribe Success/Fail
image
image

- Logged User Dashboard Notifications Tab
image
image
image
image

Logged user campaign modal
image
image

### To Reproduce

This is a design ticket

### Expected Behavior

New Design to be applied

### Which browser are you using? (if relevant)

Chrome

### PR DESCRIPTION
**Pull request by RalitsaIlieva**
Closes #1550 (news subscription)

## Motivation and context

There is new design that should be implemented.

## Screenshots:

image
image
image
image
image
image
image

Before|After
---|---
Paste screenshot|Paste screenshot

### New environment variables:

- [ ] `NEW_ENV_VAR`: env var details

### New or updated dependencies:

Dependency name|Previous version|Updated version|Details
---|---|---|---
`dependency/name`|`v1.0.0`|`v2.0.0`|

### CODE REVIEW

**Review by ani-kalpachka — APPROVED** [review]

**Review by sashko9807 — CHANGES_REQUESTED** [review]

File: src/components/common/form/CheckboxField.tsx [hunk 1]
```diff
@@ -8,13 +8,15 @@ import { TranslatableField, translateError } from 'common/form/validation'
 
 export type CheckboxFieldProps = {
   name: string
+  sx?: object
```
[the remark this answers is not in the document]
Comment (RalitsaIlieva) [reply src/components/common/form/CheckboxField.tsx]: done

File: src/components/client/campaigns/CampaignDetails.tsx [hunk 2]
```diff
@@ -133,29 +163,8 @@ export default function CampaignDetails({ campaign }: Props) {
...
-                  />
-                </Tooltip>
-              ) : (
-                ''
-              )}
-            </Typography>
-            <CampaignFinanceSummary campaign={campaign} expenses={totalExpenses ?? 0} />
-          </Grid>
-          {(expensesList?.length || canEditCampaign) && (
+        {expensesList?.length ||
```
[the remark this answers is not in the document]
Comment (RalitsaIlieva) [reply src/components/client/campaigns/CampaignDetails.tsx]: fixed

File: src/components/client/index/IndexPage.tsx [hunk 3]
```diff
@@ -28,6 +29,7 @@ export default function IndexPage() {
       <PartnersSection />
       <TeamMembersSection />
       <JoinPodkrepiBgSection />
+      <SubscriptionSection />
```
[the remark this answers is not in the document]
Comment (swolf86) [reply src/components/client/index/IndexPage.tsx:32]: https://www.figma.com/file/MmvFKzUv6yE5U2wrOpWtwS/Podkrepi.bg?type=design&node-id=3904%3A13406&mode=design&t=i8Eql4jYENDpTGgA-1

1. under what have we achieved?
Съществуващата секция “Какво постигнахме досега” да се допълни с функционалност за абониране
2. under Who is behind Podkrepi.bg?

[the remark this answers is not in the document]
Comment (RalitsaIlieva) [reply src/components/client/index/IndexPage.tsx:32]: image

[the remark this answers is not in the document]
Comment (RalitsaIlieva) [reply src/components/client/index/IndexPage.tsx:32]: image

[the remark this answers is not in the document]
Comment (RalitsaIlieva) [reply src/components/client/index/IndexPage.tsx:32]: There are 4 positions of the Subscribe for news according to the design in Figma. If something changed, please, update it on time

**Comment by ani-kalpachka** [comment]
@sashko9807 The comments seem to be fixed. Could you please check and confirm?

**Review by sashko9807 — APPROVED** [review]
Approving this in order to not block functionality tests in staging environment .  Though the commit still needs some polishing before deploying to production.

**Pull request merged**